test/utils/TransactionSender.py

```diff
@@ -37,7 +28,19 @@ async def send_transaction(self, calls, signers, nonce=None, max_fee=0):
             else:    
                 signatures += list(signer.sign(message_hash))
 
-        return await self.account.__execute__(call_inputs, calldata, nonce).invoke(signature=signatures)
+        return await self.account.__execute__(call_array, calldata, nonce).invoke(signature=signatures)
+
+def from_call_to_call_array(calls):
+    call_array = []
```

Comment:
nitpick: there are no real arrays in Python, `[]` represents lists

Reply:
Yes but that's to match the naming used in the Cairo contract.